aspnetcore/blazor/tooling.md

```diff
@@ -104,68 +116,53 @@ Select **Next**.
 
 * Select **Create**.
 
-* Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app.
-
-:::moniker range="< aspnetcore-8.0"
-
-When running a hosted Blazor WebAssembly [solution](#visual-studio-solution-file-sln) in Visual Studio, the startup project of the solution is the **:::no-loc text="Server":::** project.
-
-:::moniker-end
-
-For more information on trusting the ASP.NET Core HTTPS development certificate, see <xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos>.
-
-:::moniker range="< aspnetcore-8.0"
-
-> [!IMPORTANT]
-> When executing a hosted Blazor WebAssembly app, run the app from the solution's **:::no-loc text="Server":::** project.
->
-> When the app is launched, only the `Properties/launchSettings.json` file in the :::no-loc text="Server"::: project is used.
-
-:::moniker-end
-
 :::zone-end
 
-:::zone pivot="linux-macos"
+:::zone pivot="vsc"
 
-To create a Blazor app on Linux or macOS, use the following guidance:
+[Visual Studio Code](https://code.visualstudio.com) is an open source, cross-platform Integrated Development Environment (IDE) that can be used to develop Blazor apps.
 
-Use the [.NET command-line interface (CLI)](/dotnet/core/tools/) to execute commands in a command shell.
+Install the latest version of [Visual Studio Code](https://code.visualstudio.com/Download) for your platform.
 
-Install the latest version of the [.NET Core SDK](https://dotnet.microsoft.com/download). If you previously installed the SDK, you can determine your installed version by executing the following command:
+The Visual Studio Code (VS Code) instructions for ASP.NET Core development in this tutorial use the [.NET CLI](/dotnet/core/tools/), which is part of the .NET SDK. .NET CLI commands are issued in VS Code's integrated [**Terminal**](https://code.visualstudio.com/docs/editor/integrated-terminal), which is a [PowerShell command shell](/powershell/). The **Terminal** is opened by selecting **New Terminal** from the **Terminal** menu in the menu bar.
 
-```dotnetcli
+Install the [.NET SDK](https://dotnet.microsoft.com/download). If you previously installed one or more .NET SDKs and want to see your active version, execute the following command in the **Terminal**:
+
+```powershell
```

Comment:
Out of curiosity, why do we prefer `powershell` to `dotnetcli` for these commands?

Reply:
The **Terminal** in VSC is a PowerShell prompt by default (or via a system default). They can change it to a command shell with the little drop down ...

![image](https://github.com/dotnet/AspNetCore.Docs/assets/1622880/08f8f2d9-e725-4a82-bad3-a460a6ba9390)

I'll change these to .NET CLI because that's what the commands are, and then I'll make an update to the text to say that these .NET CLI commands work at either a command prompt or a PowerShell prompt, either of which can be selected by the user with a cross-link to the VSC docs section on it.

I'll also make this change over in the Blazor tutorial because it also uses PS for these at the moment.

Reply:
***Done!*** 👍 ... Will be on the next commit.